backend/src/test/java/ai/verta/modeldb/CommitTest.java

```diff
@@ -1619,4 +1620,80 @@ public void getCommitEnvironmentBlobListTest() throws NoSuchAlgorithmException {
 
     LOGGER.info("List commit environment blob test end................................");
   }
+
+  @Test
+  public void checkRepoUpdatedTimeWithCeateDeleteCommitTest() throws ModelDBException {
```

Comment:
Create

Reply:
right now the name is a good indicator of what the test does , but it will be helpful to add a comment explaining what the test is expected to do.

Reply:
Added Doc comment